site/zenodo_rdm/config.py

```diff
@@ -272,6 +273,30 @@
         "source": "/record/<pid_value>/export/<export_format>",
         "target": record_export_view,
     },
+    "redirect_legacy_record_export_view_hx": {
+        "source": "/records/<pid_value>/export/hx",
+        "target": legacy_record_export_view,
+    },
+    "redirect_legacy_record_export_view_dcite4": {
+        "source": "/records/<pid_value>/export/dcite4",
+        "target": legacy_record_export_view,
+    },
+    "redirect_legacy_record_export_view_xd": {
+        "source": "/records/<pid_value>/export/xd",
+        "target": legacy_record_export_view,
+    },
+    "redirect_legacy_record_export_view_geojson": {
+        "source": "/records/<pid_value>/export/geojson",
+        "target": legacy_record_export_view,
+    },
+    "redirect_legacy_record_export_view_xm": {
+        "source": "/records/<pid_value>/export/xm",
+        "target": legacy_record_export_view,
+    },
+    "redirect_legacy_record_export_view_dcat": {
+        "source": "/records/<pid_value>/export/dcat",
+        "target": legacy_record_export_view,
+    },
```

Comment:
minor: it feels a bit duplicated to have both the `ZENODO_RECORD_EXPORTERS_LEGACY` config, which contains the mapping, and these entries. Would it make sense to "generate" the dictionary entries from config:

```python
**{
    f"redirect_leagcy_record_export_view_{key}": {
        "source": f"/records/<pid_value>/export/{key}"
        "target": legacy_record_export_view,
    } for key in ZENODO_RECORD_EXPORTERS_LEGACY
}
```

...or make some sort of utility function generator, to avoid having `ZENODO_RECORD_EXPORTERS_LEGACY`:

```python
**{
    f"redirect_leagcy_record_export_view_{key}": {
        "source": f"/records/<pid_value>/export/{key}"
        "target": legacy_record_export_view(key, val),  # this would return a function that does the mapping
    } for key, val in.items()
}
```